documentation/docs/guides/experimental-features.md

```diff
@@ -2,57 +2,54 @@
 
+
+## Available Experimental Features
+
 :::note
-Experimental features are new capabilities that are still being tested and refined. While they can provide additional functionality, they may be less stable than standard features.
+The list of experimental features may change as Goose development progresses. Some features may be promoted to stable features, while others might be modified or removed.This section will be updated with specific experimental features as they become available
 :::
 
-## Enabling Experimental Features
-
-To access experimental features, run:
-
-```sh
-goose configure
-```
-
-Select "Goose Settings" from the menu:
-
-```sh
-┌ goose-configure
-│
-◆ What would you like to configure?
-| ○ Configure Providers
-| ○ Add Extension
-| ○ Toggle Extensions
-| ○ Remove Extension
-// highlight-next-line
-| ● Goose Settings (Set the Goose Mode, Tool Output, Experiment and more)
-└
-```
-
-Then select "Toggle Experiment" option in the menu:
-   ```sh
-   ┌   goose-configure 
-   │
-   ◇  What would you like to configure?
-   │  Goose Settings 
-   │
-   ◆  What setting would you like to configure?
-   │  ○ Goose Mode 
-   │  ○ Tool Output 
-   // highlight-next-line
-   │  ● Toggle Experiment (Enable or disable an experiment feature)
-   └  
+### Ollama Tool Shim
+
+The Ollama tool shim is an experimental feature that enables tool calling capabilities for language models that don't natively support tool calling (like DeepSeek). It works by instructing the primary model to output json for intended tool usage, the interpretive model uses ollama structured outputs to translate the primary model's message into valid json, and then that json is translated into valid tool calls to be invoked.
+
+
+#### How to use the Ollama Tool Shim
+
+1. Make sure you have [Ollama](https://ollama.com/download) installed and running
+2. The default interpreter model is `mistral-nemo`, if you want to proceed with this, you have to pull it from ollama server by running:
+
+   ```bash
+   ollama pull mistral-nemo
    ```
+3. If you want to use a different model, make sure to pull it first from the Ollama server. Then override the default interpreter model using the `GOOSE_TOOLSHIM_OLLAMA_MODEL` environment variable. For example, to use the `llama3.2` model, run:
 
-## Available Experimental Features
+   ```bash
+   ollama pull llama3.2
+   ```
+   Then,
 
-:::note
-There are no experimental features at this time!
-:::
+   ```bash
+   GOOSE_TOOLSHIM_OLLAMA_MODEL=llama3.2 
+   ```
+
+4. For optimal performance, run the Ollama server with an increased context length:
+   ```bash
+   OLLAMA_CONTEXT_LENGTH=32768 ollama serve
+   ```
+
+5. Enable the tool shim by setting the `GOOSE_TOOLSHIM` environment variable:
+
+   ```bash
+   GOOSE_TOOLSHIM=1 
```

Comment:
you don't need to change or update this..I'm just asking for my own knowledge..what does the 1 represent..is it like a true or false value..like if it's set to 1 then it's enabled?

Reply:
Yes it is.

Reply:
oh okay thank you